lib/node/NodeMainTemplate.runtime.js

```diff
@@ -18,7 +18,7 @@ module.exports = function() {
 		return Promise.resolve(update);
 	}
 
-	function hotDisposeChunk(chunkId) {
+	function hotDisposeChunk(chunkId) { //eslint-disable-line no-unused-vars
```

Comment:
For each of the `hotDisposeChunk` and other methods tagged with this, it is to tell the linter that the method is actually going to be used even though it is not used inside of this file because of how its injected